### PR TITLE
Nazat/feature/sms counter

### PR DESCRIPTION
**Pull request by Nazar-kob**

### CODE REVIEW

File: backend/app/router/order.py [hunk 1]
```diff
@@ -64,6 +68,23 @@ def change_status_order(
...
+    elif data.new_status == m.OrderStatus.can_not_complete.value:
+        message = MESSAGE_TEXT[settings.SMS_LANGUAGE][
+            m.OrderStatus.can_not_complete
+        ].format(business.name, business.phone_number, order_id)
+
+    if message is not None:
+        business.sms_used += 1
+        db.commit()
+        is_send = send_sms(message, order.phone_number.number)
+        # TODO what to do if message is not sent
```
Comment (Taraslysak) [inline backend/app/router/order.py:86]: Option 1. Give some feedback to user which changed the order status to let him know that smth gone wrong. HE may then try to reach customer in some other way (make a call?)
Option 2. Make a loop of some kind, which would have access  to all unsent sms and try to re-send them

Comment (Taraslysak) [reply backend/app/router/order.py:86]: IMHO, option 1 is easier to implement and would be more suitable for small businesses this project is aimed on. But this needs to be agreed with the customer

Comment (Taraslysak) [reply backend/app/router/order.py:86]: And in option 2 we would still need some mechanic to inform users in case sending sms failed some number of times

**Review by Taraslysak — COMMENTED** [review]
Nice

File: backend/app/config.py [hunk 2]
```diff
@@ -16,6 +16,8 @@ class Settings(BaseSettings):
     API_TOKEN_UP_SENDER: str = ""
     USERNAME_UP_SENDER: str = ""
     STATIC_FOLDER: str = "static"
+    SMS_LANGUAGE: str = "EN"
+    FRONTEND_URL: str = ""
```
Comment (SvyatoslavArtymovych) [inline backend/app/config.py:20]: Set localhost as default

File: backend/app/service/up_sender.py [hunk 3]
```diff
@@ -21,7 +21,7 @@ def send_sms(text_sms: str, phone_number: str) -> bool:
         "Data": {
             "Message": f"{text_sms}",
             "Recipients": [{"Phone": f"{phone_number}"}],
-            "Settings": {"Sender": "fish market"},
+            "Settings": {"Sender": "fish market"},  # TODO business name
```
Comment (SvyatoslavArtymovych) [inline backend/app/service/up_sender.py:24]: TODO

File: backend/sample.env [hunk 4]
```diff
@@ -21,3 +21,10 @@ USERNAME_UP_SENDER=<username>
 
 # add to backup uploaded images
 DATA_FOLDERS_TO_BACKUP=/app/static/market
+
+
+# we have to type of messages in English and in Hebrew so value of keys can be just "EN" or "IL" and no more keys
+SMS_LANGUAGE="EN"
+
+# use it for creation link on market ,note: without slash in the end
+FRONTEND_URL='http://127.0.0.1:8088'
```
Comment (SvyatoslavArtymovych) [inline backend/sample.env:30]: No newline at end of file

**Review by SvyatoslavArtymovych — APPROVED** [review]

**Pull request merged**